ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -76,6 +76,19 @@ spec:
     # - fileName: OadpOperatorStatus.yaml
     #   policyName: "subscriptions-policy"
     #
+    # NMState operator is used for IPsec configuration with NMState
+    # The use of NMState Operator is recommended for MNO clusters only due to resource consumption
+    # - fileName: NMStateSubscriptionNS.yaml
+    #   policyName: "subscriptions-policy"
+    # - fileName: NMStateSubscriptionOperGroup.yaml
+    #   policyName: "subscriptions-policy"
+    # - fileName: NMStateSubscription.yaml
+    #   policyName: "subscriptions-policy"
+    # - fileName: NMStateOperatorStatus.yaml
+    #   policyName: "subscriptions-policy"
+    # - fileName: NMState.yaml
+    #   policyName: "subscriptions-policy"
+    #
```

Comment:
Let's move this to `group-du-3node-ranGen.yaml` and `group-du-standard-ranGen.yaml` so there's no mistaking where this operator is allowed.